<?xml version="1.0"?>
 
<launch>
  
  <arg name="debug" default="false"/>
  <!-- <arg name="namespace"     default=" "/> -->
  <arg name="frame_id" default="arm_arduino"/>

   <!-- <group ns="$(arg namespace)"> -->
      <node name="arm_arduino_node" type="arm_arduino" pkg="siar_arm" output="screen" unless="$(arg debug)">
        <param name="motor_file" value="$(find siar_arm)/cfg/mot_"/>
        <param name="angle_file" value="$(find siar_arm)/cfg/ang_"/>
        <param name="max_pan_rate" value="1"/>
        <param name="max_tilt_rate" value="1"/>
        <param name="rate" value="10"/>
        <param name="frame_id" value="$(arg frame_id)"/>
      </node>
      <node name="arm_arduino_node" type="arm_arduino" pkg="siar_arm" output="screen" if="$(arg debug)" launch-prefix="xterm -e gdb --args">
        <param name="motor_file" value="$(find siar_arm)/cfg/mot_"/>
        <param name="angle_file" value="$(find siar_arm)/cfg/ang_"/>
        <param name="max_pan_rate" value="1"/>
        <param name="max_tilt_rate" value="1"/>
        <param name="rate" value="10"/>
        <param name="frame_id" value="$(arg frame_id"/>
      </node>
    <!-- </group> -->

</launch>
